py3status/modules/gpmdp.py

```diff
@@ -4,55 +4,125 @@
 
 Configuration parameters:
     cache_timeout:  refresh interval for this module (default 5)
-    format:         specify the items and ordering of the data in the status bar.
-                    These area 1:1 match to gpmdp-remote's options
-                    (default '♫ {info}')
+    format:         display format for this module (default '♫[ {status}]')
+    state_paused:   show this when gpmdp paused playback
+        (default '[(Paused) {song_title} - {song_artist}]')
+    state_playing:  show this when gpmdp started playback
+        (default '{song_title} - {song_artist}')
```

Comment:
state_.. already has a use at the moment and it is different.

naming is difficult ...  really I think that in this case we should manage a single format - maybe we won't we'll see in the end.